static/app/components/codecov/container/persistence.tsx

```diff
@@ -0,0 +1,83 @@
+import {captureException} from '@sentry/react';
+
+import type {CodecovContextTypes} from 'sentry/components/codecov/container/container';
+
+// Constants
+const CODECOV_PRODUCT = 'codecov';
+
+// Types
+type CodecovObjectInLocalStorage = {
```

Comment:
Currently redundant; could extend from the type in the Context component if this eventually grows or simply reuse it. Leaning towards reusing the original instead of extending.